code/tasks/CurlLinkChecker.php

```diff
@@ -28,22 +28,30 @@ public function checkLink($href) {
 		// Skip non-external links
 		if(!preg_match('/^https?[^:]*:\/\//', $href)) return null;
 
-		// Check if we have a cached result
-		$cacheKey = md5($href);
-		$result = $this->getCache()->load($cacheKey);
-		if($result !== false) return $result;
+		if (!Config::inst()->get('CurlLinkChecker', 'BypassCache')) {
```

Comment:
It would be great if we could both:

* use `underscore_casing` for config properties, and;
* define these as private statics on the class that owns them, with a `@config` phpdoc annotation

Are you still interested in this pull request @textagroup? If not, I can merge manually and make a follow up PR

Reply:
Was thinking that if we change this to private static we would also need a setter method to change this property, I would prefer to stay with using the config system so it stays as a setting similar to how you white list codes

Reply:
@textagroup I think the config system is enough. It's good practice to define the private statics for the config API on the class that owns them, tagged with `@config` as a sort of default value